packages/is-shallow-equal/src/index.js

```diff
@@ -13,10 +13,10 @@ export { default as isShallowEqualArrays } from './arrays';
 
 /**
  * Returns true if the two arrays or objects are shallow equal, or false
- * otherwise.
+ * otherwise. Also handles primitives and falsey values, just in case.
  *
- * @param {any[]|ComparableObject} a First object or array to compare.
- * @param {any[]|ComparableObject} b Second object or array to compare.
```

Comment:
This function does already work with anything, and it's useful to allow it to receive a falsey value

Reply:
I slightly disagree -- the object comparison is only called after we know that the arguments are objects. I think that function does expect them to be objects. This top-level function decides which comparison function to use based on the input type -- if the input is anything other than an array or object, it'll fall back to strict equal, which is what allows primitives to work as input